CodePush.js

```diff
@@ -273,7 +274,11 @@ async function syncInternal(options = {}, syncStatusChangeCallback, downloadProg
             if (resolvedInstallMode == CodePush.InstallMode.ON_NEXT_RESTART) {
               log("Update is installed and will be run on the next app restart.");
             } else {
-              log("Update is installed and will be run when the app next resumes.");
+              if (syncOptions.minimumBackgroundDuration > 0) {
```

Comment:
Can we Math.floor() this value before using it and logging to the user since we are not supporting decimal values?